### PR TITLE
Feature awards seeding + removing nominee_count field and adding user_id to unique constraint for awards_performances

### PR DESCRIPTION
**Pull request by feng-eric**
- Basic seeding into awards table and awards category table
- Added user_id to unique constraint to awards_performances tuple
- Removed nominee_count field from awards_performances
- Refactored existing routes to accommodate these changes 
- Tested nominating performance for multiple awards, tested nominating different performances for the same award, tested finalizing a performance for an award, tested attempting to finalize an award that is already finalized, which will throw an error.
- NOTE: Schema changes to this PR

### CODE REVIEW

**Comment by feng-eric** [comment]
Not sure why it's showing changes from the other PR I made. You can just look at changes in prisma/seed.js @thepenguinco

**Comment by thepenguinco** [comment]
> Not sure why it's showing changes from the other PR I made. You can just look at changes in prisma/seed.js @thepenguinco

Uh, are those changes supposed to be there?

**Comment by feng-eric** [comment]
> > Not sure why it's showing changes from the other PR I made. You can just look at changes in prisma/seed.js @thepenguinco
> 
> Uh, are those changes supposed to be there?

I already merged them in lol They're already on staging

File: prisma/seed.js [hunk 1]
```diff
@@ -282,6 +284,69 @@ async function dataSeed() {
...
+        id: award['id'],
+      },
+      update: {
+        title: award['title'],
+        is_category: award['is_category'],
+      },
+      create: {
+        id: award['id'],
+        title: award['title'],
+        is_category: award['is_category'],
```
Comment (OustanDing) [inline prisma/seed.js:324]: Not a big deal but we can just use `award.id`, `award.title`, and `award.is_category` here.

**Review by OustanDing — APPROVED** [review]
LGTM. Thanks for putting this up so quickly @feng-eric !

**Review by thepenguinco — APPROVED** [review]
LGTM.

**Review by OustanDing — APPROVED** [review]
New endpoint LGTM too

**Review by thepenguinco — APPROVED** [review]

**Pull request merged**